src/ui/edit_field.c

```diff
@@ -449,6 +449,13 @@ void edit_field_replace(Edit_field *edit_field, const char *text)
     }
 }
 
+void edit_field_append(Edit_field *edit_field, const char *text)
+{
+    size_t n = strlen(text);
+    edit_field->buffer = strcat(edit_field->buffer, text);
```

Comment:
Please don't use `strcat` it's prone to buffer overflows. Please use something like `strncat`.